teps/0074-deprecate-pipelineresources.md

```diff
@@ -192,25 +190,31 @@ These images are versioned alongside Tekton Pipelines, regardless of whether the
 The images are:
 
 * [git-init](https://github.com/tektoncd/pipeline/tree/main/cmd/git-init)
+  * Used in the catalog Tasks "git-clone" and "git-batch-merge"
 * [pullrequest-init](https://github.com/tektoncd/pipeline/tree/main/cmd/pullrequest-init)
+  * Used in the "pull-request" catalog Task, intended as a replacement for the pullrequest PipelineResource
 * [kubeconfigwriter](https://github.com/tektoncd/pipeline/tree/main/cmd/kubeconfigwriter)
+  * Used in the "kubeconfig-creator" catalog Task, intended as a replacement for the cluster PipelineResource
+* [imagedigestexporter](https://github.com/tektoncd/pipeline/tree/main/cmd/imagedigestexporter)
+  * No longer used in any catalog Tasks
 
-(Not sure if we'd consider [imagedigestexporter](https://github.com/tektoncd/pipeline/tree/main/cmd/imagedigestexporter)
-one of these or not?)
+If we deprecate PipelineResources, it doesn't make sense to keep these in the Tekton Pipelines codebase.
 
-If we deprecate PipelineResources, it probably doesn't make sense to keep these in the Tekton Pipelines codebase, so
-the proposal is to create a new repo `tektoncd/images` and move the code for these images there, where they can be
-maintained and released separately from Tekton Pipelines. This would also create a place where we could put other
-images that we decide we want to maintain and publish for catalog tasks.
+As part of [TEP-0079](./0079-tekton-catalog-support-tiers.md), the "git-clone" and "git-batch-merge" Tasks (plus a few others)
+will be moved to a verified catalog maintained by the Tekton org, and the community catalog will be archived.
+Code for the "git-init" image will be moved into a separate repo.
```

Comment:
which separate repo? in tektoncd or tektoncd-catalog org?

Reply:
tektoncd-catalog/git-clone; updated